src/main/java/de/caritas/cob/userservice/api/workflow/enquirynotification/scheduler/EnquiryNotificationScheduler.java

```diff
@@ -19,10 +20,13 @@ public class EnquiryNotificationScheduler {
   @Value("${enquiry.open.notification.enabled}")
   private Boolean enquiryNotificationsEnabled;
 
+  private final @NonNull TenantContextProvider tenantContextProvider;
```

Comment:
I would rather refactor a little in TenantContextProvider to use TECHNICAL_TENANT_ID from TenantContext and not from TenantResolverService